frontends/web/src/components/forms/checkbox.module.css

```diff
@@ -31,21 +31,21 @@
     top: -1px;
 }
```

Comment:
nit: there is another border color here that uses `rgb(220, 220, 220)`

Reply:
I kept this since we don't have a variable for this color. I think we can keep it for now (since I think this color works for both light and dark mode) and then in a future pr we can make it a variable.